Lib/test/test_enum.py

```diff
@@ -864,13 +864,13 @@ class Grades(int, Enum):
             C = 3
             D = 2
             F = 0
-        self.assertEqual(f'{self.Grades.B}', 'B')
+        self.assertEqual(f'{Grades.B}', 'B')
 
     @unittest.skipIf(
             python_version >= (3, 12),
             'mixin-format now uses member instead of member.value',
             )
-    def test_mixin_format_warning(self):
+    def test_mixin_format_warning_member(self):
```

Comment:
I preferred renaming the other method with `_future`, but it doesn’t really matter!

Reply:
This is now removed from `test_enum`.